.github/workflows/BuildImage.yml

```diff
@@ -7,6 +7,7 @@ env:
   ENDPOINT: "linuxserver/mods" #don't modify
   BASEIMAGE: "swag" #replace
   MODNAME: "auto-uptime-kuma" #replace
+  MULTI_ARCH: "true" #set to false if not needed
```

Comment:
This should be false as the Dockerfile is not multi-arch and both arches get the same exact files